# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../clutter-gtk/compile-fix.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

diff -Nur clutter-gtk-0.10.8-orig/clutter-gtk/Makefile.am clutter-gtk-0.10.8/clutter-gtk/Makefile.am
--- clutter-gtk-0.10.8-orig/clutter-gtk/Makefile.am	2010-12-05 18:20:14.528050530 +0100
+++ clutter-gtk-0.10.8/clutter-gtk/Makefile.am	2010-12-05 18:21:26.240056101 +0100
@@ -48,7 +48,7 @@
 	$(AM_V_GEN)$(INTROSPECTION_SCANNER) -v \
 		--namespace GtkClutter --nsversion=@CLUTTER_GTK_API_VERSION@ \
 		--strip-prefix=GtkClutter \
-		$(INCLUDES) $(AM_CPPFLAGS) \
+		$(INCLUDES) $(AM_CPPFLAGS) $(GTK_CFLAGS) \
 		--add-include-path=$(srcdir) --add-include=path=. \
 		--include=Clutter-1.0 \
 		--include=GdkPixbuf-2.0 \
